### PR TITLE
initial version

### PR DESCRIPTION
**Pull request by danielcsant**

### CODE REVIEW

File: src/main/scala/com/stratio/receiver/RabbitMQInputDStream.scala [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+  private def receive() {
+
+    try {
+      val factory: ConnectionFactory = new ConnectionFactory
+      factory.setHost(rabbitMQHost)
+      factory.setPort(rabbitMQPort)
+      val connection: Connection = factory.newConnection
+      val channel: Channel = connection.createChannel
+
+      var queueName = ""
```
Comment (gserranojc) [inline src/main/scala/com/stratio/receiver/RabbitMQInputDStream.scala]: you must use Val, you could create a function that return the name of the queue

File: src/main/scala/com/stratio/receiver/RabbitMQInputDStream.scala [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+      channel.queueDeclare().getQueue()
+    }else{
+      rabbitMQQueueName.get
+    }
+  }
+
+  /** Create a socket connection and receive data until receiver is stopped */
+  private def receive() {
+
+    try {
```
Comment (sgomezg) [inline src/main/scala/com/stratio/receiver/RabbitMQInputDStream.scala]: I think this try would be more readable if you change by something like:

for {
  factory <- getConnectionFactory // Try[ConnectionFactory]
  connection <- factory.newConnection // Try[Connection]
  channel <- connection.createChannel
} yield {
  (connection,channel)
}
...


http://danielwestheide.com/blog/2012/12/26/the-neophytes-guide-to-scala-part-6-error-handling-with-try.html

File: README.md [hunk 3]
```diff
@@ -1,2 +1,86 @@
 # RabbitMQ-Receiver
-RabbitMQ Spark Streaming receiver
+
+RabbitMQ-Receiver is a library that allows the user to read data with [Apache Spark](https://spark.apache.org/)
+from [RabbitMQ](https://www.rabbitmq.com/).
+
+## Requirements
+
+This library requires Spark 1.3+, Scala 10+, RabbitMQ 3.5+
```
Comment (sgomezg) [inline README.md]: Scala 2.10+

File: pom.xml [hunk 4]
```diff
@@ -0,0 +1,147 @@
...
+    <licenses>
+        <license>
+            <name>The Apache Software License, Version 2.0</name>
+            <url>http://www.apache.org/licenses/LICENSE-2.0.txt</url>
+        </license>
+    </licenses>
+
+    <issueManagement>
+        <system>GitHub</system>
+        <url>https://github.com/Stratio/sparkta/issues</url>
```
Comment (sgomezg) [inline pom.xml]: Check the url

**Comment by sgomezg** [comment]
@danielcsant well done! Some minor comments :)

**Comment by sgomezg** [comment]
Very nice! LGTM and merging!

**Pull request merged**